### ISSUE TITLE
bug: cannot set dynatrace_http_monitor follow_redirects to false

### ISSUE THREAD

**Issue by aslafy-z** [dynatrace-oss/terraform-provider-dynatrace#231]
**Describe the bug**
`dynatrace_http_monitor.script.request.configuration.follow_redirects = false` is not saved.

**To Reproduce**
1. Create a synthetic with `script.request.configuration.follow_redirects = false`
2. Terraform apply
3. Follow redirects is set to true in Dynatrace API but False in the terraform state

**Expected behavior**
Follow redirects is set to false in Dynatrace API and terraform state

**Additional context**

I've seen an announcement with v1.25.0 related to booleans and the way they were saved. This bug might be related.

Tested with 1.9 - 1.27

**Comment by kvaidy01** [comment]
I am facing a similar issue.

**Comment by ggreen05** [comment]
I am also facing this issue on v1.17.0, which wasn't facing issues prior to this. This issue started sometime between March 29th and April 4th (March 29th was our last successful run and April 4th was when the issue appeared).

The code will have the setting as false, in the plan and state it will show as false but when it runs the value is set as true in Dynatrace. When doing a new plan it detects the drift from the state and says it will change the value from true to false but when run it remains true.

I've attempted to recreate via the API to identify if the API is the root cause using both POST requests for new http monitors and PUT requests on existing http monitors and the value updates to false as expected.

**Comment by kvaidy01** [comment]
Can we have an update on this please when possible ?

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Hello everyone, sorry for the late response. 

Root cause for this issue is a behavioral change on the REST API side. We didn't get a response back yet from PM, but about three weeks ago we noticed that, when omitting that property, the REST API started to assume `true` as the result value (instead of `false` like it used to before).
As a temporary workaround we chose to ignore that property in the provider - hoping that the original behavior will get re-instated on the REST API, and as a matter of fact assuming that the odds any of our users is setting that to `false` is very, very low.

Release v1.29.0 (targeted for mid next week) will address that API change properly. The default value (when not specified) for `follow_redirects`  remains `false` - no schema change to be expected. In other words, the solution we should have come up with in the first place has been implemented now. Apologies for the confusion.

best regards,
Reinhard
